test/lendingPool.js

```diff
@@ -1,103 +1,184 @@
 const { expect } = require("chai");
 const { ethers, network } = require("hardhat");
-const { sign, getContract, deployAll } = require('../scripts/utils')
+const { sign, deployAll } = require('../scripts/utils');
 
-const eth1 = "1000000000000000000"
-const eth01 = "100000000000000000"
+const ONE_ETH = "1000000000000000000";
+const ONE_TENTH_OF_AN_ETH = "100000000000000000";
 
 const userAddress = "0x71a15Ac12ee91BF7c83D08506f3a3588143898B5"
 const nftAddress = "0xf5de760f2e916647fd766B4AD9E85ff943cE3A2b"
-const startMaxDailyBorrows = eth1
+const startMaxDailyBorrows = ONE_ETH
+const SECONDS_PER_DAY = 24 * 60 * 60
+const DAYS_PER_YEAR = 365;
+const INTEREST_WEI_PER_ETH_PER_YEAR = 0.8e18;
+const DEADLINE = Math.round(Date.now() / 1000) + 1000;
+const PRICE = ONE_TENTH_OF_AN_ETH;
 
 describe("LendingPool", function () {
-    it("oracle", async function () {
-        const [owner, oracle] = await ethers.getSigners();
-        const { lendingPool } = await deployAll(oracle.address, eth01, nftAddress, startMaxDailyBorrows, "TubbyLoan", "TL", "1209600", "25367833587")
-        const price = 1
-        const deadline = Math.round(Date.now() / 1000) + 1000;
-        const nftContract = await lendingPool.nftContract()
-        const signature = await sign(oracle, price, deadline, nftContract)
-        await lendingPool.setMaxPrice(eth1) // 1eth
-        await lendingPool.checkOracle(price, deadline, signature.v, signature.r, signature.s)
-    })
-
-    it("basic usage", async function () {
-        const [owner, oracle, liquidator] = await ethers.getSigners();
-        const { lendingPool, factory } = await deployAll(oracle.address, eth1, nftAddress, startMaxDailyBorrows, "TubbyLoan", "TL", "1209600", "25367833587")
-        const price = eth01 // 0.1eth
-        const deadline = Math.round(Date.now() / 1000) + 1000;
-        const nftContract = await lendingPool.nftContract()
+    let owner;
+    let oracle;
+    let liquidator;
+    let factory;
+    let lendingPool;
+    let nft;
+    let user;
+
+    this.beforeAll(async function () {
```

Comment:
++on @0xngmi 's suggestion to add a mocked NFT here - tho forking a specific block number from Goerli does the same, if a bit lazier